contracts/src/v0.8/llo-feeds/v0.5.0/configuration/Configurator.sol

```diff
@@ -161,6 +161,7 @@ contract Configurator is IConfigurator, ConfirmedOwner, TypeAndVersionInterface,
 
     ConfigurationState memory configurationState = s_configurationStates[configId];
     if (
+      predecessorConfigDigest == bytes32(0) ||
```

Comment:
should we add a test for this case?

Reply:
There is one, in this PR

Reply:
ok, it wasn't 100% clear when I first looked at the test